README.md

```diff
@@ -108,7 +108,7 @@ As a general example, we will use the [python-helloworld](https://github.com/dba
 To run `RepoAuditor`, we can enter the following in the command-line:
 
 ```shell
-uvx RepoAuditor --include GitHub \
+uvx RepoAuditor --includes GitHub \

```

Comment:
I can see how we have a problem here as `--include` makes more sense on the command line (because it can be provided multiple times) while `includes` makes more sense in the config file because it is a key for a value that is an array. If we have to make a choice, I'd say that we should favor the command line over the config file, as people have to use the command line but need to opt in to using the config file.  
  
However, I'm wondering if there is functionality in `typer-config` that addresses this discrepancy as we can't be the first people to run into this problem.